features/support/apiREST.js

```diff
@@ -14,13 +14,11 @@ ApiREST.prototype.init = function (world) {
 ApiREST.prototype.disconnect = function () {};
 
 ApiREST.prototype.pathApi = function (path) {
-  var basePath = '/api';
-
-  if (apiVersion) {
-    basePath += '/' + apiVersion;
-  }
+  return config.url + '/api/1.0/' + path;
```

Comment:
Because only Kuzzle knows its API version, and functional tests are clients to Kuzzle. 
To get the API version, the only solution would be to perform a "serverInfo", but that would mean starting to use Kuzzle before effectively testing it.
So... hardcoded. :-)